redhat/release/update-to-head.sh

```diff
@@ -67,27 +72,16 @@ if [[ -d redhat/patches ]]; then
   git apply redhat/patches/*
 fi
 
-# RHTAP writes its pipeline files to the root of ${redhat_ref}
-# Fetch those from origin and apply them to the the release branch
-# since we just wiped out our local copy with the upstream ref.
-git fetch origin $redhat_ref
-git checkout origin/$redhat_ref .tekton
-
 # Move overlays to root
 if [[ -d redhat/overlays ]]; then
-  if [[ -f redhat/overlays/Dockerfile ]]; then
-      git rm Dockerfile
-  fi
```

Comment:
This if statement is needed, the script will error when trying to replace the Dockerfile

Reply:
@JasonPowr I thought you'd catch this! :) It actually isn't required since the next line has been changed to a simple `mv` instead of `git mv`. As a result, the contents of the existing `./Dockerfile` are simply modified, and the subsequent `git add` later in the script just adds the changes to the commit. I think it's a little cleaner this way because it's easier to see exactly what was changed.

Reply:
Ah no worries so, my mistake :)